GANDLF/losses/__init__.py

```diff
@@ -38,3 +37,26 @@
     "focal": FocalLoss,
     "dc_focal": DC_Focal,
 }
+
+
+def get_loss(params: dict) -> object:
+    """
+    Function to get the loss definition.
+
+    Args:
+        params (dict): The parameters' dictionary.
+
+    Returns:
+        loss (object): The loss definition.
+    """
+    # TODO This check looks like legacy code, should we have it?
+
+    if isinstance(params["loss_function"], dict):
+        chosen_loss = list(params["loss_function"].keys())[0].lower()
```

Comment:
Perhaps, I am not fully aware of that functionality and which config convention does it adhere to, but maybe we can think of standardizing the dict format for loss config. I mean, if the loss_function is indeed dict, then we require it to be something like this: 
```yaml
- "loss_function":
    - "name" : <loss_name>,
    - "config" : <dict_with_custom_loss_config>
````
That would allow us to use the new refactored losses in clean way, thoughts?

Reply:
Agreed - should this be a separate PR to the master?

Reply:
Yeah, can be. I can pack it with other warnings corrections.

Reply:
Actually, I see that at master branch, the loss is retrieved in the code, there is not "getter" function, so perhaps we can wait with that change when the lightning will be integrated?

Reply:
Okay, sure.